### PR TITLE
krakend flexible configuration

### PR DESCRIPTION
**Pull request by rizzza**
Flesh out more the krakend flexible configuration and add the loadbalancerapi endpoints.

```bash
# Ensure local dev container for loadbalancer-api is up and running and exposed on port 7608
curl  localhost:7608/v1/tenant/8263b19b-6aea-4dd7-a2cd-ffa7efe513e9/loadbalancers
{
  "version": "v1",
  "kind": "loadBalancersList",
  "load_balancers": []
}

# spin up local api-gateway
make run-local

# hit api gateway
curl http://localhost:8080/v1/tenant/8263b19b-6aea-4dd7-a2cd-ffa7efe513e9/load_balancers
{
  "version": "v1",
  "kind": "loadBalancersList",
  "load_balancers": []
}
```

### CODE REVIEW

File: config/settings/prod/loadbalancerapi_v1.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "backend_hosts": [
```
[the remark this answers is not in the document]
Comment (rizzza) [reply config/settings/prod/loadbalancerapi_v1.json:2]: indeed

**Comment by rizzza** [comment]
> Minus CI not passing this is inline with what I was looking for. @JAORMX I know you have opinions and we have multiple repos at this point. How does this fit with the things you have found talking with the krakend team and what you are thinking?

@andy-v-h mentioned we could use `buildkite` for CI, since it now has access to ghcr.io

**Review by JAORMX — APPROVED** [review]
Let's give it a try

**Comment by JAORMX** [comment]
The main thing I'm looking for is having a minimal way to inject the backend services.

**Comment by JAORMX** [comment]
After some thought, let's take this approach to the main repo https://github.com/infratographer/api-gateway-endpoints and discuss it with the folks working on the API Gateway. That approach is more inclusive and respectful of people's previous work than creating a competing approach.

**Pull request closed without merging**